packages/samples/headless-commerce-react/playwright.config.ts

```diff
@@ -0,0 +1,33 @@
+import {defineConfig, devices} from '@playwright/test';
+
+export default defineConfig({
+  testDir: './e2e',
+  fullyParallel: true,
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 0,
+  workers: process.env.CI ? 1 : undefined,
+  reporter: 'html',
+  use: {
+    baseURL: 'http://localhost:3000',
+    trace: 'on-first-retry',
+  },
+  projects: [
+    {
+      name: 'chromium',
+      use: {...devices['Desktop Chrome']},
+    },
+    {
+      name: 'firefox',
+      use: {...devices['Desktop Firefox']},
+    },
+    {
+      name: 'webkit',
+      use: {...devices['Desktop Safari']},
+    },
+  ],
+  webServer: {
+    command: 'npx ws -d dist -p 3000',
+    port: 3000,
```

Comment:
Again this seems redundant: why must we specify the port here if we pass it in the command?